modules/apps/portal/portal-tika/src/main/java/com/liferay/portal/tika/internal/extract/TextExtractorImpl.java

```diff
@@ -110,6 +116,12 @@ public String extractText(InputStream inputStream, int maxStringLength) {
 		return text;
 	}
 
+	@Activate
+	@Modified
+	protected void activate(Map<String, Object> properties) {
+		TikaConfigurationUtil.updateConfiguration(properties);
```

Comment:
Hey, @MrJohn1911 let's try to find a different approach for this fix. With this change, every time the  `com.liferay.portal.tika.internal.configuration.TikaConfiguration` changes the `TikaConfigurationUtil.updateConfiguration` will run twice(`TextExtractorImpl#activate` and `TikaRawMetadataProcessor#activate`)